packages/xcm-util/asset-sort/compare-assets.ts

```diff
@@ -39,6 +39,14 @@ import {
 } from './compare-utils';
 import {interiorToArray} from '../common';
 
+/**
+ * Compares two assets of any type based on their asset IDs and fungibility.
```

Comment:
Please correct this in all the similar places.

```suggestion
 * Compares two assets of any version based on their asset IDs and fungibility.
```